portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,72 @@
+<!--<script src="https://kit.fontawesome.com/6593008338.js" crossorigin="anonymous"></script>-->
+
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Phuong Nguyen</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <script src="https://kit.fontawesome.com/6593008338.js" crossorigin="anonymous"></script>
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+    <div id="nav-home">
+      <input type="checkbox" id="check">
+      <label for="check" class="checkbtn">
+        <i class="fas fa-bars"></i>
+      </label>
+      
+      <ul>
+        <li><a class="underline" href="index.html#about">ABOUT</a></li>
+        <li><a class="underline" href="education.html">EDUCATION</a></li>
+        <li><a class="underline" href="projects.html">PROJECTS</a></li>
+        <li><a class="underline" href="contact-form.html">CONTACT</a></li>
+        <!--<div class="drop-down content"> 
```

Comment:
It's okay to have comment-outs in self exploration project, but in a collaborative project, it would be great to only keep valid code.

Reply:
I understand. I will try to remove comments-out when making pull requests next time. Thank you!